src/graphql/schema.gql

```diff
@@ -311,6 +336,16 @@ type Subscription {
   created: Int!
 }
 
+type User {
+  id: String!
+  ipfs: String
+  address: String!
+  username: String
+  bio: String
```

Comment:
Sorry missed to review on the snapshot.js PR but the fields should be:

name (instead of username)
about (instead of bio)

Also i think it should be stored as a JSON in the db to get more flexibility if we want add fields on the schema (similar than space table now)

Reply:
so the fields will be profile - JSON

and inside of it name and bio 

is that good?

Reply:
@bonustrack can't this be expressed beforehand?

Reply:
If these are not separate fields in the db table, we should at least use the tools mysql provides for json fields: https://dev.mysql.com/doc/refman/8.0/en/json-search-functions.html#operator_json-column-path
Otherwise this whole design doesn't really make sense to me and just blocks because it's unclear what should be a column and what should go into the json. I mean... we could just have tables with an id, created/updated, ipfs hash, and a json field. But we need to define that somewhere.

Reply:
@mktcode I can use JSON datatype instead of TEXT, however for the purpose of this PR, do you see us using any JSON functions to search / store values?

Reply:
Well, we should actually avoid accessing the JSON data in our SQL queries anyway. So a text field would also be ok. On the other hand, for the custom domains I found that having the domain of a space be an indexed column makes sense but now I'd have to use MySQL JSON functions to do a lookup by domain. We just need to think well about what we need as conditions in SQL queries and what can be just part of the data blob.
Fabien sent me this article: https://www.wix.engineering/post/scaling-to-100m-mysql-is-a-better-nosql
Which helped me a lot to understand the design decisions here.

> Any field that is not used as a condition in a query has been folded into a single blob field